xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachmentsinline.vm

```diff
@@ -45,7 +45,13 @@ $xwiki.ssfx.use('js/xwiki/viewers/attachments.css', true)
     ## Temporarily disabled, until we fix attachment name handling
     ## <div><label id="xwikiuploadnamelabel" for="xwikiuploadname">$services.localization.render('core.viewers.attachments.upload.filename')</label></div>
     ## <div><input id="xwikiuploadname" type="hidden" name="filename" value="" size="40"/></div>
-    <div class="fileupload-field"><label class="sr-only" for="xwikiuploadfile">$services.localization.render('core.viewers.attachments.upload.file')</label><input id="xwikiuploadfile" type="file" name="filepath" size="40" class="uploadFileInput noitems" data-max-file-size="$!escapetool.xml($xwiki.getSpacePreference('upload_maxsize'))" /></div>
+    <div class="fileupload-field">
+      #if ($services.attachment.configuration.areSummariesEnabled())
+        <label for="xwikiuploadcomment" class="sr-only">$services.localization.render('core.viewers.attachments.upload.comment')</label>
+        <input type="text" id="xwikiuploadcomment" class="uploadCommentInput" name="comment" placeholder="$services.localization.render('core.viewers.attachments.upload.comment')" title="$services.localization.render('core.comment.tooltip')" maxlength="1023" />
```

Comment:
Placeholders must not be used as replacement for a label. See also [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder):

> Effective placeholder text includes a word or short phrase that hints at the expected data type, not an explanation or prompt. The placeholder must not be used instead of a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). As the placeholder is not visible if the value of the form control is not null, using placeholder instead of a `<label>` for a prompt harms usability and accessibility.

Instead, the placeholder could provide an example for the data that should be entered.

Further, I don't understand why you're using `core.comment.tooltip` as tooltip. "Enter a brief description of your changes" doesn't make sense here.